backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/bottomsheet/internal/BottomSheetContent.kt

```diff
@@ -0,0 +1,40 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.bottomsheet.internal
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun bottomSheetContent(
```

Comment:
Gotchu!

Reply:
Well, i cannot make the name uppercase as it is having a return type.

Reply:
Also, i don't know from where we will be passing a modifier as it is a extracted internal function which will no more be modified anywhere?

Reply:
ah just spotted it. take a look at how the `BpkBottomSheetHandle` is done - this should be a standard composable function and then set like `{ BottomSheetContent ()}` rather than returning the lambda as that's against compose guidelines

Reply:
Done!